### ISSUE TITLE
Two seemingly related internal errors

### ISSUE THREAD

**Issue by m0davis** [agda/agda#2257]
Location of the error: src/full/Agda/TypeChecking/Telescope.hs:317

``` agda
record R : Set₁ where
  foo : Set
  foo = ∀ {f} → {!!}

  field
    bar : Set
```

Location of the error: src/full/Agda/TypeChecking/Rules/Term.hs:159

``` agda
record R (A : Set) : Set₁ where
  field
    a : A
    F : ∀ {f} → Set

  foo : Set
  foo = ∀ {f} → {!!}

  field
    bar : Set
```

**Comment by nad** [comment]
Bisection suggests that the problem (at least the first one) was introduced by @andreasabel in b7e340690351eb2713d3d835fa4b4277b9cfc158: "Fixed #707: do not type check interaction meta twice."

**Comment by andreasabel** [comment]
I made these errors go away, but a proper fix of #434 would be the correct thing to do here.
